SwitchItem/SwitchItem.js

```diff
@@ -100,32 +100,29 @@ const SwitchItemBase = kind({
 	},
 
 	computed: {
-		icon: ({css, icon}) => (icon ? <Icon small className={css.icon}>{icon}</Icon> : null)
+		icon: ({css, icon}) => (icon ? <Cell shrink component={Icon} small className={css.icon}>{icon}</Cell> : null)
 	},
 
 	render: ({children, css, icon, offText, onText, selected, ...rest}) => {
 
 		return (
-			<div {...rest} css={css}>
+			<Row align="center" {...rest}>
 				{icon}
-				<div
-					className={css.text}
-				>
+				<Cell className={css.text}>
 					{children}
-				</div>
-				<div className={css.label}>
+				</Cell>
+				<Cell shrink className={css.label}>
 					{selected ? onText : offText}
-				</div>
-				<AgateSwitch selected={selected} className={css.switchIcon} />
-			</div>
+				</Cell>
+				<Cell shrink component={Switch} selected={selected} className={css.switchIcon} />
+			</Row>
 		);
 	}
 });
 
-
 const SwitchItemDecorator = compose(
-	Toggleable({toggleProp: 'onTap'}),
-	Touchable,
+	Toggleable({toggleProp: 'onClick'}),
+	// Touchable,
```

Comment:
Do we need to remove `Touchable`?

Reply:
I did because it didn't seem to _help_, but it also didn't really seem to _hurt_ either? I don't feel strongly about its presence.